activesupport/lib/active_support/key_generator.rb

```diff
@@ -33,7 +33,7 @@ def initialize(secret, options = {})
       @hash_digest_class = options[:hash_digest_class] || self.class.hash_digest_class
     end
 
-    # Returns a derived key suitable for use.  The default key_size is chosen
+    # Returns a derived key suitable for use.  The default +key_size+ is chosen
     # to be compatible with the default settings of ActiveSupport::MessageVerifier.
     # i.e. OpenSSL::Digest::SHA1#block_length
```

Comment:
Since `OpenSSL::Digest::SHA1#block_length` is external, it won't be highlighted (unlike `ActiveSupport::MessageVerifier`).  Perhaps we can address that here too?

```suggestion
    # i.e. <tt>OpenSSL::Digest::SHA1#block_length</tt>
```

(Note that we need to use `<tt>` because of the `#`.)

Reply:
Done!